packages/lib-classifier/src/store/utils/session.spec.js

```diff
@@ -87,7 +87,7 @@ describe('Store utils > sessionUtils', function () {
 
     it('should call generateSessionID', function () {
       sessionUtils.getSessionID()
-      expect(generateSessionIDStub.calledTwice).to.be.true()
+      expect(generateSessionIDStub).to.have.been.calledTwice()
```

Comment:
Why is the session ID generated twice when you call `getSessionID()`?

Reply:
Looking through [`session.js`](https://github.com/zooniverse/front-end-monorepo/blob/ab843296fb2b42fc8fd42024455c85e469d29383/packages/lib-classifier/src/store/utils/session.js), the code should only call `generateSessionID` once.

Reply:
Fixed in 6814400.